backends/cuda/ceed-cuda-qfunction-load.cpp

```diff
@@ -51,9 +51,9 @@ extern "C" int CeedCudaBuildQFunction(CeedQFunction qf) {
   CeedQFunction_Cuda *data;
   ierr = CeedQFunctionGetData(qf, (void **)&data); CeedChk(ierr);
   // QFunction is built
-  if (!data->qFunctionSource)
+  if (data->qFunction)
     return 0;
-  
+
```

Comment:
```suggestion
  if (!data->QFunctionSource)
    return CeedError(ceed, "No QFunction source or CUfunction provided.");
```

would require moving up lines 128,129